src/main/java/xyz/nucleoid/bedwars/game/active/ItemGeneratorPools.java

```diff
@@ -0,0 +1,50 @@
+package xyz.nucleoid.bedwars.game.active;
+
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import xyz.nucleoid.bedwars.game.config.BwConfig;
+import xyz.nucleoid.bedwars.game.config.GeneratorConfig;
+
+import java.util.ArrayList;
+
+public class ItemGeneratorPools {
+    public final ItemGeneratorPool TEAM_LVL_1;
```

Comment:
```suggestion
    public final ItemGeneratorPool teamLevel1;
```
Since these are no longer static final constants 🙂